### PR TITLE
fix(deploy): add required env-vars and fix port forwarding

### PR DESCRIPTION
**Pull request by debloper**
In docker-compose integrated stack.

Changes for minishift stack will be updated in to existing PR #2342

### CODE REVIEW

**Comment by fabric8cd** [comment]
@debloper fabric8/fabric8-ui:SNAPSHOT-PR-2377-1 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2377-fabric8-planner.badger.fabric8.io

File: docker-compose.f8ui.yml [hunk 1]
```diff
@@ -6,6 +6,10 @@ services:
     image: fabric8-planner-platform
     # command: npm run test:unit
     environment:
+      FABRIC8_REALM: "fabric8-test"
+      FABRIC8_WIT_API_URL: "http://localhost:8080/api/"
+      FABRIC8_AUTH_API_URL: "http://localhost:8089/"
```
Comment (jarifibrahim) [inline docker-compose.f8ui.yml]: This should be `http://localhost:8089/api`

Comment (debloper) [reply docker-compose.f8ui.yml]: good catch, fix coming

**Comment by fabric8cd** [comment]
@debloper fabric8/fabric8-ui:SNAPSHOT-PR-2377-2 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2377-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@debloper fabric8/fabric8-ui:SNAPSHOT-PR-2377-3 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2377-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@debloper fabric8/fabric8-ui:SNAPSHOT-PR-2377-5 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2377-fabric8-planner.badger.fabric8.io

**Pull request merged**